src/components/WideButton/__tests__/WideButton.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+
+import WideButton from '../WideButton';
+
+jest.mock('react-router-dom', () => ({
+  Link: 'Link'
+}));
+
+describe('COMPONENT - WideButton', () => {
+  it('renders Link if to prop is provided', () => {
+    const component = create(<WideButton variant="frontend" to="/frontend" />);
+
+    expect(component.toJSON()).toMatchSnapshot();
+  });
+
+  it('render button if onClick is provided', () => {
```

Comment:
renders Link if `onClick` prop is provided -> put the prop name `onClick` in between ` sign